docs/04-operation-guides/troubleshooting/svls-05-serverless-periodically-restarting.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Serverless periodically restarting
+---
+
+
+## Symptom
+
+Serverless restarting every 10 minutes when reconciling git-sourced functions.
+
+## Cause
+
+Function controller is polling for changes in referenced git repositories. If you have a lot of git-sourced functions and they were deployed together in approximately the same time their git sources will be checked-out in a synchronised pulse (every 10 minutes). If you happen to reference large repositories (multi-repositories) there will be rhythmical high demand on cpu and I/O resources necessary to checkout repositories that may cause function controller to crash and restart.
```

Comment:
```suggestion
Function controller is polling for changes in referenced git repositories. If you have a lot of git-sourced Functions and they were deployed together at approximately the same time, their git sources will be checked out in a synchronised pulse (every 10 minutes). If you happen to reference large repositories (multi-repositories), there will be rhythmical high demand on cpu and I/O resources necessary to checkout repositories. This may cause Function controller to crash and restart.
```